tools/openms/FeatureFinderSuperHirn.xml

```diff
@@ -105,6 +105,6 @@ python3 '$__tool_directory__/fill_ctd.py' '@EXECUTABLE@.ctd' '$args_json' '$hard
   <help><![CDATA[Finds mass spectrometric features in mass spectra.
 
 
-For more information, visit https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/release/2.6.0/html/UTILS_FeatureFinderSuperHirn.html]]></help>
+For more information, visit http://www.openms.de/doxygen/nightly/html/UTILS_FeatureFinderSuperHirn.html]]></help>
```

Comment:
Maybe we should migrate this help into a macro. This will reduce the diff and also gives us a handle to update the lnks more easily manually.